src/couch_replicator_manager.erl

```diff
@@ -134,9 +134,14 @@ replication_error(#rep{id = {BaseId, _} = RepId}, Error) ->
 continue(#rep{doc_id = null}) ->
     {true, no_owner};
 continue(#rep{id = RepId}) ->
-    Owner = gen_server:call(?MODULE, {owner, RepId}, infinity),
-    {node() == Owner, Owner}.
-
+    case rep_state(RepId) of
+    nil ->
+        {false, nonode};
+    #rep_state{rep = #rep{db_name = DbName, doc_id = DocId}} ->
+        Node = node(),
+        Owner = owner(DbName, DocId, [Node | nodes()]),
```

Comment:
(The idea being that, eventually, all nodes would have rescanned with the same [node()|nodes()] value if we're tracking it in state. It could well be just the same as you are doing).